catalyst/contrib/nn/criterion/contrastive.py

```diff
@@ -135,8 +135,77 @@ def forward(self, embeddings_pred, embeddings_true) -> torch.Tensor:
         return loss
 
 
+class BarlowTwinsLoss(nn.Module):
```

Comment:
could you please also drop a couple of tests for this module?
like https://github.com/catalyst-team/catalyst/blob/master/tests/catalyst/metrics/functional/test_accuracy.py#L48#L83

Reply:
just to check the correctness of the implementation ;)

Reply:
👍

Reply:
[Done](ContrastivePairwiseEmbeddingLoss)

Reply:
Where should I put the tests?
I think here `tests/catalyst/contrib/nn/test_criterion.py` what do you think?

Reply:
LGTM 👍

Reply:
I think these [tests](https://github.com/Nimrais/catalyst/blob/barlow-twins/tests/catalyst/contrib/nn/test_criterion.py#L32-L108) are good enough.